### DISCUSSION TITLE
Need guidance on understanding the logs from mapserver

### DISCUSSION THREAD

**Discussion by mizzPpY** [camptocamp/docker-mapserver#524] [Q&A]
Hi, 
I have a docker image that using this mapserver as the base and package my own Mapfile in /etc/mapserver, it is deploy in a kubernetes cluster.
I am debugging some issue which all the return tile images are empty image; when I check the logs, it seems all my request is returning status 200. Below is some sample logs from the pod:

> 6082563 9b7c5c4d-e601-4458-8a0a-0f6c87f32653 127.0.0.6 - - [31/May/2023:08:55:36 +0000] "GET /mapserver/?map=/etc/mapserver/custom.map&Service=WMS&mode=tile&layers=SomeLayerName&tile=2479+1279+12&someparam=10813 HTTP/1.1" 200 582 "-" "-"
> 6085537 e8a9d361-df93-48cd-9390-335011904b38 127.0.0.6 - - [31/May/2023:08:55:36 +0000] "GET /mapserver/?map=/etc/mapserver/custom.map&Service=WMS&mode=tile&layers=SomeLayerName&tile=2474+1281+12&someparam=10813 HTTP/1.1" 200 582 "-" "-"

If I understand it correctly, the first number comes after "HTTP/1.1" is the status code? May I know what is the next number means, also what is the last 2 dashes means? Is there any guide on how to understand mapserver logs?

Many thanks in advance.

**Comment by sbrunner** [comment]
The log format is the following:
`%{us}T %{X-Request-Id}i %h %l %u %t \"%r\" %>s %O "%{Referer}i" "%{User-Agent}i"`
Apache documentation: https://httpd.apache.org/docs/current/logs.html#accesslog
Then it's the response size, then the Referer and User-Agent request headers.
